packages/flutter/android/build.gradle.production

```diff
@@ -0,0 +1,59 @@
+// The Android Gradle Plugin builds the native code with the Android NDK.
+
+group 'com.example.flutter_breez_liquid'
+version '0.2.1'
```

Comment:
version script should update this version

Reply:
Done [00bfede](https://github.com/breez/breez-sdk-liquid/pull/488/commits/00bfede5f84fbd40e966b444db74a14a88fa0695)